crates/diffbot_lib/src/github/github_api.rs

```diff
@@ -221,7 +212,7 @@ pub async fn download_file<S: AsRef<str>>(
     path.push(&target.sha);
     path.set_extension("dmi");
 
-    tokio::fs::create_dir_all(path.parent().unwrap()).await?;
+    async_fs::create_dir_all(path.parent().unwrap()).await?;
```

Comment:
Why do you use this instead of tokio::fs?

Reply:
shared lib, don't want another tokio